CHANGES.txt

```diff
@@ -83,8 +83,10 @@ Changes
     SELECT id FROM t2
 
 - Added new "password" authentication method which is available for connections
-  via the Postgres Protocol. This method allows clients to authenticate using
-  a valid database user and its password.
+  via the Postgres Protocol and HTTP. This method allows clients to authenticate
+  using a valid database user and its password. For HTTP the ``X-User`` header,
+  used to provide a username, has been deprecated in favor of the the standard
+  HTTP Authorization header with the Basic Authentication Scheme.
```

Comment:
``Authorization``

Reply:
It's called Authentication: https://en.wikipedia.org/wiki/Basic_access_authentication